src/components/shared/basic/ReleaseProcess/TechnicalIntegration/index.tsx

```diff
@@ -134,13 +134,18 @@ export default function TechnicalIntegration() {
 
   const handleCheckedUserProfiles = (checked: boolean, item: userRolesType) => {
     const isSelected = techUserProfiles?.includes(item.roleId)
+    let selectedProfiles: string[] = []
     if (!isSelected && checked) {
-      setTechUserProfiles([...techUserProfiles, item.roleId])
+      selectedProfiles = [...techUserProfiles, item.roleId]
     } else if (isSelected && !checked) {
       const oldTechUserProfiles = [...techUserProfiles]
       oldTechUserProfiles.splice(oldTechUserProfiles.indexOf(item.roleId), 1)
-      setTechUserProfiles([...oldTechUserProfiles])
+      selectedProfiles = [...oldTechUserProfiles]
     }
+    selectedProfiles?.length === 0
+      ? setEnableUserProfilesErrorMessage(true)
+      : setEnableUserProfilesErrorMessage(false)
```

Comment:
write above three lines as
`setEnableUserProfilesErrorMessage(selectedProfiles?.length === 0)`
because the result of the comparison will be true or false